ftw/http.py

```diff
@@ -84,8 +84,20 @@ def check_for_cookie(self, cookie):
                         break
                 cover_domain = cover_domain[first_non_dot:]
                 # We must parse the coverDomain to make sure its not in the suffix list
+                psl_path = os.path.dirname(__file__) + os.path.sep + \
+                'util' + os.path.sep + 'public_suffix_list.dat'
```

Comment:
they CAN be but they won't be until we decide on what those will be. I'm expected a PR from you soon, it's be great to change it in that PR so as not to make redundent work